python/cudnn/gemm/cutedsl/grouped/unfused/__init__.py

```
@@ -3,4 +3,14 @@
 
 from .api import GroupedGemmSm100, grouped_gemm_wrapper_sm100
 
-__all__ = ["GroupedGemmSm100", "grouped_gemm_wrapper_sm100"]
+__all__ = ["GroupedGemmSm100", "grouped_gemm_wrapper_sm100", "grouped_gemm_jax_sm100"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)